__tests__/database/migration/file/index.test.ts

```diff
@@ -7,6 +7,6 @@ describe('TESTING all migration file', () => {
     // Execution
 
     // Assertion
-    expect(allMigrations.length).toEqual(9);
+    expect(allMigrations.length).toEqual(16);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The number of migrations has increased from 9 to 16. This is a significant change and it's important to ensure that all these new migrations have been thoroughly tested, especially considering the potential impact on existing data. If not already done, consider adding tests for each individual migration to verify its correctness.
```diff
-    expect(allMigrations.length).toEqual(16);
+    // Suggested test structure for each migration
+    describe('Migration X', () => {
+        it('should correctly migrate data', () => {
+            // Setup: Create data in old format
+
+            // Execution: Run migration
+
+            // Assertion: Check data is in new format
+        });
+    });
```
Please replace 'X' with the actual migration version number. Repeat this for each new migration.


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
no need